kmod/patch/kpatch-patch.h

```diff
@@ -37,7 +37,7 @@ struct kpatch_patch_dynrela {
 	unsigned long type;
 	char *name;
 	char *objname;
-	int external;
+	char *sym_objname;
```

Comment:
I'd leave 'external' here, and still set it like before so that kpatch can use it.

Also, we need to try to stay backwards compatible as much as possible, so that newer kpatch-build will still work with old kernels which don't have this livepatch change.  Maybe with some ifdefs in `livepatch-patch-hook.c` like what @arges did in #573.